src/control/SConscript

```diff
@@ -127,15 +137,17 @@ def install_go_bin(denv, gosrc, libs, name, install_name):
                   get_build_tags(denv), build_bin, install_src))
     # Use the intermediate build location in order to play nicely
     # with --install-sandbox.
-    denv.Requires(build_bin, ['gurt_lib'])
```

Comment:
gurt_lib isn't actually needed here, probably as it's passed in as a lib above, or maybe because common depends on it all already.

I'm working on a test locally to verify the build and will update this PR later.